src/compute/src/memory_management/policy.rs

```diff
@@ -131,6 +133,9 @@ impl JemallocMemoryControl {
             } else {
                 tracing::info!("Successfully dumped heap profile to {}", file_name);
             }
+            let _ = unsafe {
```

Comment:
May leave some notes show that why we have to workaround for a fake `static` lifetime here.